boa/src/value/to_integer_or_infinity.rs

```diff
@@ -0,0 +1,99 @@
+use super::*;
+
+use std::convert::TryFrom;
+
+/// Represents the result of ToIntegerOrInfinity operation
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum IntegerOrInfinity {
+    Integer(isize),
```

Comment:
I think `i64` should be used here, since in 32-bit arch. `isize` is 4-bytes, and cannot fit the MAX_SAFE_INTEGER

Reply:
Yeah, I think you're right. This would also simplify some conversions down the road.